### PR TITLE
Improve visibility of lint-staged usage warning (#3076)

### PR DESCRIPTION
**Pull request by juanjoLenero**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -398,7 +398,7 @@ and add this config to your `package.json`:
   "lint-staged": {
     "*.{js,json,css}": [
       "prettier --write",
-      "git add"
+      "git add" // Warning: This breaks staging specific lines. See footnote below.
```
Comment (lydell) [inline README.md]: Hmm, this is JSON, which doesn't support comments ...

File: tests/markdown/README.md [hunk 2]
```diff
@@ -388,7 +388,7 @@ and add this config to your `package.json`:
   "lint-staged": {
     "*.{js,json,css}": [
       "prettier --write",
-      "git add"
+      "git add" // Warning: This breaks staging specific lines. See footnote below.
```
Comment (lydell) [inline tests/markdown/README.md]: You've edited the input to a test without changing its expected output, which is why Travis fails. I'm not sure we should update the test, though.

**Comment by lydell** [comment]
Thanks for putting together a PR! Really appreciate it. However, I'm not sure this is any better.

File: README.md (hunk 1 above)
Comment (juanjoLenero) [reply README.md]: Well isn't this embarassing 😅

**Comment by juanjoLenero** [comment]
I guess you are right, the warning is already there. I could maybe just add bold markup to it or prefix it with "**warning:**" if you think that would help. Otherwise I don't think there's much that can be done.

File: tests/markdown/README.md (hunk 2 above)
Comment (azz) [inline tests/markdown/README.md]: You don't want to update this file. @ikatyang maybe we should rename it so people don't accidentally find it :P

**Pull request closed without merging**